restapi/config.go

```diff
@@ -46,11 +45,17 @@ var (
 
 	// TLSRedirect console tls redirect rule
 	TLSRedirect = "on"
-
-	// SessionDuration cookie validity duration
-	SessionDuration = 45 * time.Minute
 )
 
+// GetConsoleCookieDurationInSeconds returns the default session duration for the console cookie in seconds (default 3600 seconds)
+func GetConsoleCookieDurationInSeconds() int {
+	duration, err := strconv.Atoi(env.Get(ConsoleSTSDurationSeconds, "3600"))
+	if err != nil {
+		duration = 3600
+	}
+	return duration
+}
+
```

Comment:
```suggestion
// GetConsoleCookieDuration returns the default session duration for the console cookie in seconds (defaults to 1h)
func GetConsoleCookieDuration() time.Duration {
	duration, err := time.ParseDuration(env.Get(ConsoleSTSDurationSeconds, "1h"))
	if err != nil {
		duration = 1 * time.hour
	}
	return duration
}
```

It is better done this way @Alevsk we can break away from the current implementation.